duneggd/SubDetector/SandInnerVolume.py

```diff
@@ -28,9 +28,13 @@ def build_tracker(self, main_lv, geom):
         # if "STT" not in self.builders:
         #     print("STT builder not found")
         #     return  
+        #print(self.builders)
```

Comment:
See above comment about commented out code.

Reply:
this has been cleared.